model/CSVDialog.cc

```diff
@@ -34,6 +53,111 @@ void CSVDialog::reportFromFile(const std::string& input, const std::string& outp
   reportFromCSVFile(is,of,spec);
 }
 
+// Return file name after downloading a CSV file from the web.
+std::string CSVDialog::loadWebFile(const std::string& url)
+{
+    try
+    {
+		// Parse input URL
+        boost::regex ex("(http|https)://([^/ :]+):?([^/ ]*)(/?[^ #?]*)\\x3f?([^ #]*)#?([^ ]*)");
+        boost::cmatch what;
+        regex_match(url.c_str(), what, ex);
+        
+        // The web domain 
+        auto const host = what[2];
+        
+        // Set the port type
+        string port = string(what[1].first, what[1].second);
+        if (string(what[1].first, what[1].second)=="https") port = "https"; 
+        else port = "http";
+        auto const target = string(what[4].first, what[4].second);  
+        
+        // Protocol version
+        int version;
+        if (!string(what[6].first, what[6].second).empty() && !std::strcmp("1.0", what[6].second)) version= 11;
+        else version=10;
```

Comment:
This bit of code makes no sense to me.

Reply:
It seems the web importer will only work with version=10. I don't understand why it doesn't work with version=11.